test/MSVC/hierarchical.py

```diff
@@ -46,7 +45,7 @@
 # TODO:  this is order-dependent (putting 'mssdk' second or third breaks),
 # and ideally we shouldn't need to specify the tools= list anyway.
 env = Environment(tools=['mssdk', 'msvc', 'mslink'])
-env['PCH'] = 'StdAfx.pch'
+env['PCH'] = File('StdAfx.pch')
```

Comment:
although you can use any name you want, the _default_ for current (since 2019) Visual Studio is `pch.h` and thus presumably `pch.pch`.

Reply:
O.k. I think for now, no need to change.
I wonder when they updated the de-facto standard file name.